modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,203 @@
+With Apache Tika you can search the contents of files of the types: PDF, DOCX, XLSX, PPTX, ODF, HTML, XML, EPUB, RTF, Tar, RAR, AR, CPIO, Zip, 7Zip, Gzip, BZip2 and many more... Read more: xref:{latest-ocis-version}@ocis:ROOT:deployment/services/s-list/search.adoc#tika-extractor[Search Service].
+
+[discrete]
+=== Space Templates
+
+Applying a template is a breeze. When creating a new project space, simply right-click on a folder of your choice, select "Create Space from selection" and the entire folder structure, complete with subfolders and files, will be instantly generated for a project space. Save time and ensure consistency by eliminating the need to manually create folders and subfolders every time you start a new project or onboard a new team. Space templates ensure a consistent folder structure across projects providing a standardized, efficient, and customizable approach to organizing your documents. Space Templates are only available to users with the permission to create spaces.
+
+[discrete]
+=== Custom User Roles
+
+Infinite Scale ships with the default user roles admin, space admin, user and guest. Each role has different permissions. For example, only space admins can create spaces, but people with the role "User" cannot create a space. With custom roles, you can now configure roles that fit the needs of your organization. For example, if you need custom roles for your school, you can configure the roles "Teacher" and "Pupil" and not give pupils the permission to create public links. If you are interested in creating custom roles, reach out to us on https://talk.owncloud.com/channel/infinitescale[Talk].
+
+[discrete]
+=== File Versions: Design Overhaul
+
+File versions received a design overhaul so that you can go back easily in the version history of a file. With the new design of the file versions feature, you now have even easier access to a comprehensive version history for each file. Every time a file is modified or updated, a new version is created and stored, preserving a complete timeline of changes. This allows you to track and revert to previous versions as needed, ensuring data integrity and eliminating the risk of accidental data loss.
+
+[discrete]
+=== Administration Settings
+
+This update brings a range of functions to streamline user, group, and space administration, enhancing the overall management experience. Let's explore the exciting new capabilities:
+
+[discrete]
+==== Users Administration:
+
+* Filter Chips for Groups and Role: +
+You can now conveniently filter users based on groups and roles, making it easier to locate and manage specific user segments efficiently.
```

Comment:
```suggestion
https://m2.material.io/components/chips#filter-chips[Filter chips] use tags or descriptive words to filter content. With that, you can now conveniently filter users based on groups and roles, making it easier to locate and manage specific user segments efficiently.
```
Explain the term `Filter chips`.

Reply:
I would not start explaining UI elements here. we should rather leve the word "Chip" out and only speak of "filter" if this makes it more comprehensible